mappings/net/minecraft/client/gui/widget/WidgetTextInput.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS none/bem net/minecraft/client/gui/widget/WidgetTextInput
```

Comment:
I'd say `WidgetTextField`, because they're usually referred to (not just in MC) as text fields.